lib/ravelin/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ravelin
-  VERSION = '0.1.41'
+  VERSION = '0.1.41.1'
```

Comment:
This is not SemVer, is this allowed?

Reply:
True, it should. SemVer removed the revision level. Will change it to `0.1.42`